### PR TITLE
Beverages

### PR DESCRIPTION
**Pull request by AntonPuko**

### CODE REVIEW

File: src/components/BeverageForm.js [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+    // todo replace to fetchAll when it will be implemented
+    this.props.availabilityStore.fetchMany();
+    this.props.srmStore.fetchMany({
+      orderBy: [{ column: 'hex', direction: 'desc ' }],
+    });
+    this.props.glassStore.fetchMany();
+    this.props.styleStore.fetchMany();
+  }
+
+  // todo Implement custom component for srm
```
Comment (jlkalberer) [inline src/components/BeverageForm.js]: I was going to say that a dropdown would work I'm guessing that's the native picker :/

File: src/components/BeverageForm.js [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+                </Picker>
+              )}
+            </FormField>
+            <FormField
+              initialValue={beverage.servingTemperature}
+              name="servingTemperature"
+            >
+              {(formFieldProps: FormFieldChildProps): React.Node => (
+                <Picker label="Serving Temperature" {...formFieldProps}>
+                  <Picker.Item label="Cellar" value="cellar" />
```
Comment (jlkalberer) [inline src/components/BeverageForm.js]: Isn't there an API to fetch these?

File: src/components/BeverageForm.js (hunk 1 above)
Comment (AntonPuko) [reply src/components/BeverageForm.js]: It works, I made the field with picker.. but I saw that in v1 you have custom slider

File: src/components/BeverageForm.js (hunk 2 above)
Comment (AntonPuko) [reply src/components/BeverageForm.js]: nope

**Pull request merged**